Unit test failing

Unit test is failing.

```
1) suggestion unit tests createVirtualHistoryItems handles input being null/undefined:
     AssertionError: Map {} deepEqual []
      at Context.<anonymous> (test/unit/app/renderer/lib/suggestionTest.js:43:14)
```

Change was made here https://github.com/brave/browser-laptop/commit/c738e7fd9b958a7116ed6f6743749609092b0df7

--- a/test/unit/app/renderer/lib/suggestionTest.js
+++ b/test/unit/app/renderer/lib/suggestionTest.js
@@ -39,7 +39,7 @@ describe('suggestion unit tests', function () {
 
   describe('createVirtualHistoryItems', function () {
     it('handles input being null/undefined', function () {
-      const emptyResult = Immutable.Map().toArray()
+      const emptyResult = Immutable.Map()
       assert.deepEqual(suggestion.createVirtualHistoryItems(), emptyResult)
       assert.deepEqual(suggestion.createVirtualHistoryItems(undefined), emptyResult)
       assert.deepEqual(suggestion.createVirtualHistoryItems(null), emptyResult)

